concepts/05 Widgets/TileView/15 Keyboard Support.md

```diff
@@ -40,7 +40,7 @@ You can implement a custom handler for a key using the [registerKeyHandler(key,
 
     <!--JavaScript-->
     function registerKeyHandlers () {
-        let tileView =  $("#tileViewContainer").dxTileView("instance");
+        const tileView =  $("#tileViewContainer").dxTileView("instance");
```

Comment:
This code example has been put into an include. You can replace this entire paragraph with a single line:

```
#include common-code-register-key-handler
```

Reply:
fixed